modules/rest-api/pages/rest-fts-partition-file-transfer.adoc

```diff
@@ -0,0 +1,79 @@
+= Rebalance with File Transfer
+:description: The Search-Service REST API allows new partitions to be created by file transfer.
+
+[abstract]
+{description}
+
+[#http-methods-and-uris]
+== HTTP Method and URI
+
+----
+PUT /api/managerOptions
+----
+
+[#description]
+== Description
+
+The Search Service automatically partitions its indexes across all Search nodes in the cluster, ensuring optimal distribution, following rebalance.
+
+To achieve this, in versions of Couchbase Server prior to 7.1, by default, partitions needing to be newly created were entirely _built_, on their newly assigned nodes.
+In 7.1+, by default, new partitions are created by the _transfer_ of partition files from old nodes to new nodes: this significantly enhances performance.
+This is an Enterprise-only feature, which requires all Search Service nodes _either_ to be running 7.1 or later; _or_ to be running 7.0.2, with the feature explicitly switched on.
+
+During file transfer, should an unresolvable error occur, file transfer is automatically abandoned, and _partition build_ is used instead.
+
+== Curl Syntax
+
+----
+curl -X PUT http://<ip-address-or-domain-name>:8094/api/managerOptions
+  -u <username>:<password>
+  -d '{"disableFileTransferRebalance": [ "true" | "false" ]}'
+  -H "Content-type:application/json"
+----
+
+If the value specified for the key `disableFileTransferRebalance` is `false` (which is the default in 7.1+), new Search-Service partitions are created during rebalance by means of partition-file transfer.
+If the value is `true`, partitions are created by means of partition building.
```

Comment:
by means of partition building => by means of partition building from scratch over DCP connections from KV/Data service.